### DISCUSSION TITLE
How to change empty description to english or any string resource to english

### DISCUSSION THREAD

**Discussion by tungthanhss** [ng-alain/ng-alain#2471] [Q&A]
I'm using latest version 17.0.5. 
For example,I want to change text bellow to eng. I already set default lang to es-Us.
Thank you.
![image](https://github.com/ng-alain/ng-alain/assets/34281514/d043ea85-d925-407c-863f-b6fb646f6c48)

**Comment by cipchk** [comment]
Pls refer to https://ng-alain.com/docs/i18n/en?#Englishversion
